src/test/memory_test.cc

```diff
@@ -393,6 +395,72 @@ TEST_F(AllocatedMemoryTest, Release)
   CHECK_POINTER_ATTRIBUTES(ptr, cudaMemoryTypeDevice, expect_id);
 }
 
+class GrowableMemoryTest : public ::testing::Test {
+ protected:
+  // Per-test-suite set-up.
+  static void SetUpTestSuite()
+  {
+    cudaFree(0);
+    cuInit(0);
+    auto status = tc::CudaBlockManager::Create(6.0);
+    EXPECT_TRUE(status.IsOk()) << status.Message();
+  }
+
+  void TearDown() override {}
+};
+
+TEST_F(GrowableMemoryTest, AllocGPU)
```

Comment:
Can we have some kind of simple test that requests CPU memory and verifies that it gets back GPU memory (or fails)? Would be a good placeholder for testing if we eventually add CPU, and will fail if we try to make those changes in the future without updating the testing.

Reply:
Added.